src/ServiceHealthChecking/FailedRequestInsurancesCheck.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace Cego\ServiceHealthChecking;
+
+use Composer\InstalledVersions;
+use Cego\RequestInsurance\Models\RequestInsurance;
+
+class FailedRequestInsurancesCheck extends BaseHealthCheck
+{
+    /**
+     * @inheritDoc
+     */
+    protected function getDescription(): string
+    {
+        return 'Checks that failed request insurances is below threshold';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function shouldSkip(): bool
+    {
+        return ! InstalledVersions::isInstalled('cego/request-insurance') || ! config('service-health-checking.request-insurance.perform-check');
+    }
+
+    /**
+     * Gets the failed RI count
+     *
+     * @return int
+     */
+    protected function getCount(): int
+    {
+        /** @phpstan-ignore-next-line  */
+        return RequestInsurance::where('abandoned_at', null)
+            ->where('paused_at', '!=', null)
```

Comment:
For god ordens skyld, tjek også lige at completed er null

Reply:
Done